src/db.js

```diff
@@ -1,16 +1,35 @@
-const { Sequelize } = require('sequelize');
+const { Sequelize, DataTypes } = require('sequelize');
 
 // Get the database connection
 const db = new Sequelize('sqlite::memory:');
 
+//Define our models
+const User = db.define('User', {
+  username: DataTypes.STRING,
+  birthday: DataTypes.DATE,
+});
 
+const Movie = db.define('Movie', {
+  nameOfMovie:  DataTypes.STRING,
+  typeOfMovie:  DataTypes.STRING,
+  releaseDate:  DataTypes.DATE,
+});
 
+const Plant = db.define('Plant', {
+  name: DataTypes.STRING,
+  size: DataTypes.STRING,
+  color: DataTypes.STRING,
+});
```

Comment:
Theses are suppose to have their own file. try look at the demo code in the class repo for [class 03](https://github.com/codefellows/seattle-javascript-401n21/tree/main/classes/class-03/demo/api-server) this will also help with the UPDATE too